utils/queryFormatUtils.ts

```diff
@@ -150,3 +147,18 @@ export const convertSequencingReadsQuery = (query: string): string => {
 
   return query;
 };
+
+export const convertConsensusGenomesQuery = (query: string): string => {
+  return (
+    query
+      // Replace Fed variables.
+      .replace(
+        /query [\s\S]*?{/,
```

Comment:
Is there a way (even if verbose) to do this without regex, or can we add an explanatory comment for anyone who comes across this?  Also, is this worth adding a test for (or is it implicitly tested by other tests)?

Reply:
It's tested by the unit tests, I'll add a comment!